A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A32NX_NDCompass.js

```diff
@@ -626,6 +639,112 @@ class Jet_MFD_NDCompass extends Jet_NDCompass {
         if (this.selectedHeadingGroup) this.selectedHeadingGroup.setAttribute("visibility", failed ? "hidden" : "visible");
         if (this.neutralLine) this.neutralLine.setAttribute("visibility", failed ? "hidden" : "visible");
     }
+
+    /**
+     * Updates navigation aid arrows such as VOR or ADF bearings
+     */
+    updateNavAid() {
+        // Don't show arrows if ADIRS not ready
+        var failed = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum") != 2;
```

Comment:
use const not var
'''suggestion
const failed = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum") != 2;
''''